### ISSUE TITLE
How can I CUDA acceleration for the prover?

### ISSUE THREAD

**Issue by JingBoGuan** [risc0/zeth#41]
Can I use GPU to accelerate the prove?I am a beginner of the zkp.

**Comment by tzerrell** [comment]
Yes. Turn on the `cuda` feature flag for the`risc0-zkvm` crate when you run the prover. If you're run of our examples, they have a `cuda` feature that passes through to `risc0-zkvm`, so you can turn on the `cuda` feature at the command line when running, e.g.
```
cargo run -r -F cuda
```

**Comment by JingBoGuan** [comment]
> Yes. Turn on the `cuda` feature flag for the`risc0-zkvm` crate when you run the prover. Most of our examples have a `cuda` feature that passes through to `risc0-zkvm`, so you can turn on the `cuda` feature at the command line when running, e.g.
> 
> ```
> cargo run -r -F cuda
> ```

thanks,it works

**Comment by tzerrell** [comment]
Great!
